### PR TITLE
[8.0] add distinta qweb

### PR DESCRIPTION
**Pull request by scigghia**
Aggiunge un report di stampa in qweb, con le informazioni della distinta riba

### CODE REVIEW

**Comment by CiroBoxHub** [comment]
:+1:

File: l10n_it_ricevute_bancarie/__openerp__.py [hunk 1]
```diff
@@ -49,6 +49,8 @@
         "views/wizard_riba_file_export.xml",
         "views/account_config_view.xml",
         "riba_workflow.xml",
+        "report.xml",
+        "views/distinta_report.xml",
         "security/ir.model.access.csv",
     ],
     'images': [],
```
Comment (OpenCode) [inline l10n_it_ricevute_bancarie/__openerp__.py]: Please remove empty key 'images'

File: l10n_it_ricevute_bancarie/report/__init__.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+##############################################################################
```
Comment (OpenCode) [inline l10n_it_ricevute_bancarie/report/__init__.py]: Change header with this model: https://github.com/OCA/maintainer-tools/blob/master/template/module/__init__.py#L1

File: l10n_it_ricevute_bancarie/report/distinta_qweb.py [hunk 3]
```diff
@@ -0,0 +1,46 @@
+# -*- coding: utf-8 -*-
+##############################################################################
```
Comment (OpenCode) [inline l10n_it_ricevute_bancarie/report/distinta_qweb.py]: Change header with this model: https://github.com/OCA/maintainer-tools/blob/master/template/module/__init__.py#L1

File: l10n_it_ricevute_bancarie/__init__.py [hunk 4]
```diff
@@ -24,5 +24,4 @@
 #
```
Comment (OpenCode) [inline l10n_it_ricevute_bancarie/__init__.py]: Change header with this model: https://github.com/OCA/maintainer-tools/blob/master/template/module/__init__.py#L1

**Comment by OpenCode** [comment]
@scigghia @CiroBoxHub resolve review on code then :+1:

File: l10n_it_ricevute_bancarie/views/distinta_report.xml [hunk 5]
```diff
@@ -0,0 +1,124 @@
...
+        </style>
+
+        <t t-call="report.html_container">
+            <t t-foreach="docs" t-as="o">
+                <t t-call="l10n_it_ricevute_bancarie.distinta_layout">
+                    <div class="page">
+
+                        <div id="distinta_number" class="row with_bord" style="padding: 2px;">
+                            <p class="distinta_number">
+                                Distinta Ricevute Bancarie nr <span t-esc="o.name or 'Bozza'"/> del <span t-field="o.registration_date"/>
```
Comment (OpenCode) [inline l10n_it_ricevute_bancarie/views/distinta_report.xml]: Use english like others strings. Then translate them.

**Comment by paolovalier** [comment]
:+1:

File: l10n_it_ricevute_bancarie/views/distinta_report.xml [hunk 6]
```diff
@@ -0,0 +1,124 @@
...
+                                    </tr>
+                                </tbody>
+                            </table>
+                        </div> <!-- /row -->
+
+                        <div class="row">
+                            <div class="col-xs-4 pull-right">
+                                <table class="table table-condensed">
+                                    <tr class="border-black">
+                                        <td><strong>Totale</strong></td>
```
Comment (OpenCode) [inline l10n_it_ricevute_bancarie/views/distinta_report.xml]: Use english like others strings. Then translate them.

**Comment by giudalo** [comment]
:+1:

**Comment by scigghia** [comment]
tnx

**Pull request merged**